homework.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+    // task1
+// for(let i=0; i<=20;i++){
+//   console.log(i)
+// }
+
+// for(let i=12;i>=-14;i--){
+//   if(i%2!==0){
+//     console.log(i)
+//   }
+// } 
+
+// for(let i=12;i>=-14;i--){
```

Comment:
Use spaces after , and between scopes, braces. That way the code will be more readable.